android/build.gradle

```diff
@@ -63,7 +63,9 @@ dependencies {
     implementation 'androidx.annotation:annotation:1.2.0'
 
     api("com.revenuecat.purchases:purchases:$purchases_version")
-    
+    implementation("com.revenuecat.purchases:purchases-core-common:$purchases_version")
```

Comment:
to get purchases-flutter working pointing to local hybrid-common, i actually had to make both of these api... 

i guess we can test purchases-flutter using a hybrid-common snapshot version before we release this